Drop collection timeout after creating index on field not existed

**Please state your issue using the following template and, most importantly, in English.**

**Describe the bug**
l-1 case: `test_index.py::TestIndexBase::test_create_index_on_field_not_existed`
```
-------------------------------- live log call ---------------------------------
[2020-09-22 01:59:19-ERROR-milvus.client.grpc_handler]: Error: <BaseException: (code=5, message=Invalid field name)> (grpc_handler.py:44)
PASSED                                                                   [ 15%]
test_index.py::TestIndexBase::test_create_index_on_field_not_existed[get_simple_index1] 
-------------------------------- live log setup --------------------------------
[2020-09-22 01:59:19-INFO-root]: {'index_type': 'IVF_FLAT', 'metric_type': 'L2', 'params': {'nlist': 128}} (test_index.py:35)
-------------------------------- live log call ---------------------------------
[2020-09-22 01:59:19-ERROR-milvus.client.grpc_handler]: Error: <BaseException: (code=5, message=Invalid field name)> (grpc_handler.py:44)
PASSED                                                                   [ 15%]
------------------------------ live log teardown -------------------------------
[2020-09-22 02:00:19-ERROR-milvus.client.grpc_handler]: RPC error: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.DEADLINE_EXCEEDED
	details = "Deadline Exceeded"
	debug_error_string = "{"created":"@1600740019.549761853","description":"Error received from peer ipv4:10.100.20.177:19530","file":"src/core/lib/surface/call.cc","file_line":1062,"grpc_message":"Deadline Exceeded","grpc_status":4}"
>
	{'API start': '2020-09-22 01:59:19.549235', 'RPC start': '2020-09-22 01:59:19.549341', 'RPC error': '2020-09-22 02:00:19.550007'} (grpc_handler.py:58)

test_index.py::TestIndexBase::test_create_index_on_field_not_existed[get_simple_index1] ERROR [ 15%]
test_index.py::TestIndexBase::test_create_index_on_field_not_existed[get_simple_index2] 
-------------------------------- live log setup --------------------------------
[2020-09-22 02:00:49-ERROR-milvus.client.grpc_handler]: RPC error: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.DEADLINE_EXCEEDED
	details = "Deadline Exceeded"
	debug_error_string = "{"created":"@1600740049.835780278","description":"Error received from peer ipv4:10.100.20.177:19530","file":"src/core/lib/surface/call.cc","file_line":1062,"grpc_message":"Deadline Exceeded","grpc_status":4}"
>
	{'API start': '2020-09-22 02:00:19.834651', 'RPC start': '2020-09-22 02:00:19.834944', 'RPC error': '2020-09-22 02:00:49.835986'} (grpc_handler.py:58)


==================================== ERRORS ====================================
_ ERROR at teardown of TestIndexBase.test_create_index_on_field_not_existed[get_simple_index1] _

tp = <class 'grpc._channel._MultiThreadedRendezvous'>, value = None, tb = None

    def reraise(tp, value, tb=None):
        try:
            if value is None:
                value = tp()
            if value.__traceback__ is not tb:
                raise value.with_traceback(tb)
>           raise value

/usr/local/lib/python3.6/site-packages/six.py:703: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/usr/local/lib/python3.6/site-packages/six.py:703: in reraise
    raise value
/usr/local/lib/python3.6/site-packages/six.py:703: in reraise
    raise value
/usr/local/lib/python3.6/site-packages/allure_commons/_allure.py:181: in __call__
    return self._fixture_function(*args, **kwargs)
conftest.py:138: in teardown
    connect.drop_collection(collection_name, timeout=delete_timeout)
/usr/local/lib/python3.6/site-packages/milvus/client/stub.py:35: in inner
    return func(self, *args, **kwargs)
/usr/local/lib/python3.6/site-packages/milvus/client/stub.py:347: in drop_collection
    return handler.drop_collection(collection_name, timeout)
/usr/local/lib/python3.6/site-packages/milvus/client/grpc_handler.py:59: in handler
    raise e
/usr/local/lib/python3.6/site-packages/milvus/client/grpc_handler.py:42: in handler
    return func(self, *args, **kwargs)
/usr/local/lib/python3.6/site-packages/milvus/client/grpc_handler.py:373: in drop_collection
    status = rf.result()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.DEADLINE_EXCEEDED
	details = "Deadline Exc...77:19530","file":"src/core/lib/surface/call.cc","file_line":1062,"grpc_message":"Deadline Exceeded","grpc_status":4}"
>
timeout = None

    def result(self, timeout=None):
        """Returns the result of the computation or raises its exception.
    
        See grpc.Future.result for the full API contract.
        """
        with self._state.condition:
            timed_out = _common.wait(self._state.condition.wait,
                                     self._is_complete,
                                     timeout=timeout)
            if timed_out:
                raise grpc.FutureTimeoutError()
            else:
                if self._state.code is grpc.StatusCode.OK:
                    return self._state.response
                elif self._state.cancelled:
                    raise grpc.FutureCancelledError()
                else:
>                   raise self
E                   grpc._channel._MultiThreadedRendezvous: <_MultiThreadedRendezvous of RPC that terminated with:
E                   	status = StatusCode.DEADLINE_EXCEEDED
E                   	details = "Deadline Exceeded"
E                   	debug_error_string = "{"created":"@1600740019.549761853","description":"Error received from peer ipv4:10.100.20.177:19530","file":"src/core/lib/surface/call.cc","file_line":1062,"grpc_message":"Deadline Exceeded","grpc_status":4}"
E                   >

/usr/local/lib/python3.6/site-packages/grpc/_channel.py:625: _MultiThreadedRendezvous
------------------------------ Captured log setup ------------------------------
[2020-09-22 01:59:19-INFO-root]: {'index_type': 'IVF_FLAT', 'metric_type': 'L2', 'params': {'nlist': 128}} (test_index.py:35)
----------------------------- Captured stderr call -----------------------------
Error: <BaseException: (code=5, message=Invalid field name)>
------------------------------ Captured log call -------------------------------
[2020-09-22 01:59:19-ERROR-milvus.client.grpc_handler]: Error: <BaseException: (code=5, message=Invalid field name)> (grpc_handler.py:44)
--------------------------- Captured stderr teardown ---------------------------
RPC error: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.DEADLINE_EXCEEDED
	details = "Deadline Exceeded"
	debug_error_string = "{"created":"@1600740019.549761853","description":"Error received from peer ipv4:10.100.20.177:19530","file":"src/core/lib/surface/call.cc","file_line":1062,"grpc_message":"Deadline Exceeded","grpc_status":4}"
>
	{'API start': '2020-09-22 01:59:19.549235', 'RPC start': '2020-09-22 01:59:19.549341', 'RPC error': '2020-09-22 02:00:19.550007'}
---------------------------- Captured log teardown -----------------------------
[2020-09-22 02:00:19-ERROR-milvus.client.grpc_handler]: RPC error: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.DEADLINE_EXCEEDED
	details = "Deadline Exceeded"
	debug_error_string = "{"created":"@1600740019.549761853","description":"Error received from peer ipv4:10.100.20.177:19530","file":"src/core/lib/surface/call.cc","file_line":1062,"grpc_message":"Deadline Exceeded","grpc_status":4}"
>
	{'API start': '2020-09-22 01:59:19.549235', 'RPC start': '2020-09-22 01:59:19.549341', 'RPC error': '2020-09-22 02:00:19.550007'} (grpc_handler.py:58)
! _pytest.outcomes.Exit: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.DEADLINE_EXCEEDED
	details = "Deadline Exceeded"
	debug_error_string = "{"created":"@1600740049.835780278","description":"Error received from peer ipv4:10.100.20.177:19530","file":"src/core/lib/surface/call.cc","file_line":1062,"grpc_message":"Deadline Exceeded","grpc_status":4}"
> !
```
**Steps/Code to reproduce behavior**
http://internal.zilliz.com:18080/jenkins/job/milvus-ci/job/PR-3820/5/artifact/_artifacts/centos7/gpu/milvus_gpu_mysql_dev_test.log
http://internal.zilliz.com:18080/jenkins/job/milvus-ci/job/0.11.0/147/artifact/_artifacts/centos7/gpu/milvus_gpu_mysql_dev_test.log
**Expected behavior**
A clear and concise description of what you expected to happen.

**Method of installation**
- [ ] Docker/cpu
- [x] Docker/gpu
- [ ] Build from source

**Environment details**
- Hardware/Softwars conditions (OS, CPU, GPU, Memory)


- Milvus version (master or released version)


**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

```yaml
paste-file-content-here
```

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here.

--- a/core/src/server/delivery/request/DropCollectionReq.cpp
+++ b/core/src/server/delivery/request/DropCollectionReq.cpp
@@ -43,9 +43,6 @@ DropCollectionReq::OnExecute() {
 
         STATUS_CHECK(DBWrapper::DB()->DropCollection(collection_name_));
 
-        /* flush to trigger CleanUpFilesWithTTL */
-        STATUS_CHECK(DBWrapper::DB()->Flush());
-
         rc.ElapseFromBegin("done");
     } catch (std::exception& ex) {
         return Status(SERVER_UNEXPECTED_ERROR, ex.what());

